site/content/blogs/release-v131.en.md

```diff
@@ -36,4 +36,27 @@ Copilot v1.31 brings big enhancements to help you develop more flexibly and effi
 
 ## Better task failure logs
 
+Before Copilot v1.31, in order to find out why your ECS tasks stopped, you'd have to navigate to AWS Console -> ECS -> Service -> Stopped Tasks -> Stopped Reason.
+
+With this enhancement, `copilot [noun] deploy` will now display the ECS task stopped reasons within your CloudFormation deployment progress tracker. Copilot will show the two most recent task failures during deployments of your Load Balanced Web Service, Backend Service and Worker Services.
+
+```console
+  - An ECS service to run and maintain your tasks in the environment cluster
+    Deployments                                                                                                              
+               Revision  Rollout        Desired  Running  Failed  Pending                                                            
+      PRIMARY  11        [in progress]  1        0        1       0                                                                  
+      ACTIVE   8         [completed]    1        1        0       0                                                                  
+    Latest 2 stopped tasks                                                                                                   
+      TaskId    CurrentStatus   DesiredStatus                                                                                        
+      6b1d6e32  DEPROVISIONING  STOPPED                                                                                              
+      9802d212  STOPPED         STOPPED                                                                                              
+                                                                                                                                     
+    ✘ Latest 2 tasks stopped reason                                                                                 
+      - [6b1d6e32,9802d212]: Essential container in task exited                                                                      
+                                                                                                                                     
+    Troubleshoot task stopped reason                                                                                         
+      1. You can run `copilot svc logs --previous` to see the logs of the last stopped task.                                
+      2. You can visit this article https://repost.aws/knowledge-center/ecs-task-stopped.          
```

Comment:
Sorry to add this nit so late in the process. 
```suggestion
      2. You can visit this article: https://repost.aws/knowledge-center/ecs-task-stopped.          
```
Can add that colon to the output?